jsapp/xlform/src/view.rowDetail.coffee

```diff
@@ -377,6 +377,13 @@ module.exports = do ->
     afterRender: ->
       @listenForCheckboxChange()
 
+  viewRowDetail.DetailViewMixins.read_only =
+    html: ->
+      @$el.addClass("card__settings__fields--active")
+      viewRowDetail.Templates.checkbox @cid, @model.key, _t("Read Only")
```

Comment:
It looks like the existing convention from neighboring lines (e.g. "Default response", "Mandatory response") is to capitalize the first word only. Please try to follow what the existing code is doing; of course, if there's no logical pattern, then use your best judgement :ok_hand: